command.c

```diff
@@ -30,8 +30,8 @@ static void process_command(char *string)
 	else if (starts_with(string, "set wheels_separation "))
 		set_wheels_separation(
 		    parse_float(string, RECEIVE_BUFFER_SIZE, 2));
-	else if (starts_with(string, "set max_linear_speed "))
-		set_max_linear_speed(
+	else if (starts_with(string, "set run_max_linear_speed "))
+		set_run_max_linear_speed(
```

Comment:
I would call it `max_linear_speed` (I think "run" is implied, as the run speed is higher than the search speed and the search speed is limited by other factors, not a calibratable robot-dependent parameter.

Reply:
Linear speed limit (?). Otherwise okay.

Reply:
9f70efa23b